examples/MNIST/output.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+        "extra": {},
+        "features": {
+            "image": [
+                0,
+                .
```

Comment:
```suggestion
```

Reply:
It looks like the docs failed to generate due to the `.`'s, let's just remove them.

Reply:
I was reading the wrong warning, looks like this one is file, my bad